<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addCtr</class>
 <widget class="QDialog" name="Dialog_addCtr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Adding Country</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_addCtr {
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #f857a6, stop:1 #ff5858);
}</string>
  </property>
  <widget class="QLabel" name="label_activity">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_activity {
color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Adding Country</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_nameCtr"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_nameCap"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_numsEps"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>222</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="styleSheet">
       <string notr="true">#pushButton_cancel {
width: 100px;
height: 35px;
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="styleSheet">
       <string notr="true">#pushButton_OK {
width: 100px;
height: 37px;
border-style: solid;
border-width: 2px;
border-color: #ffffff;
color: #ffffff;
background-color: #2E85D3;
font-weight: bold;
}
#pushButton_OK:hover {
background-color: #7fbdf5;
border-color: #b8ffbd;
}
</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_UpCont">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>67</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_UpCont {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>Which continent</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ctry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_Ctry {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>Country Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_nums">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_nums {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>No. of Episodes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cap">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_cap {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>Capital</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nameContUp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>219</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
